File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/nivel-reprezentare-comisii-report-filter.model.ts
import { JsonDateConverter } from "@app/shared/json-mapper";
import { JsonObject, JsonProperty } from "json2typescript";

@JsonObject
export class NivelReprezentareComisiiReportFilterModel {

    @JsonProperty("institutieId", Number)
    public institutieId: number = null;

    @JsonProperty("dataExpirareMandatDeLa", JsonDateConverter)
    public dataExpirareMandatDeLa: Date = null;

    @JsonProperty("dataExpirareMandatPanaLa", JsonDateConverter)
    public dataExpirareMandatPanaLa: Date = null;

}

File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/NivelReprezentareComisiiReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class NivelReprezentareComisiiReportFilterModel {
	
	private Long institutieId;
	private Date dataExpirareMandatDeLa;
	private Date dataExpirareMandatPanaLa;
	
	public Long getInstitutieId() {
		return institutieId;
	}
	public void setInstitutieId(Long institutieId) {
		this.institutieId = institutieId;
	}
	public Date getDataExpirareMandatDeLa() {
		return dataExpirareMandatDeLa;
	}
	public void setDataExpirareMandatDeLa(Date dataExpirareMandatDeLa) {
		this.dataExpirareMandatDeLa = dataExpirareMandatDeLa;
	}
	public Date getDataExpirareMandatPanaLa() {
		return dataExpirareMandatPanaLa;
	}
	public void setDataExpirareMandatPanaLa(Date dataExpirareMandatPanaLa) {
		this.dataExpirareMandatPanaLa = dataExpirareMandatPanaLa;
	}
	
}
